com.expidusos.calculator.yml

```diff
@@ -0,0 +1,36 @@
+app-id: com.expidusos.calculator
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+separate-locales: false
+command: calculator
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+modules:
+  - name: FileManager
+    buildsystem: simple
+    build-commands:
+      - cp -R bundle /app/expidus_calculator
+      - mkdir -p /app/bin && ln -s /app/expidus_calculator/calculator /app/bin/calculator
+      - chmod +x /app/bin/calculator
+      - mkdir -p /app/expidus_calculator/lib && mv /app/expidus_calculator/lib/*.so /app/expidus_calculator/lib
```

Comment:
why do you do that?
source and destination are the same.

Reply:
The manifest was copied from the file manager app and in https://github.com/flathub/com.expidusos.file_manager/pull/3, there was a weird issue where this had to be done. The issue is not present with the calculator for some reason despite using the same exact workflow steps. I just haven't removed the unnecessary move.

Reply:
Fixed